testSrc/io/flutter/util/DartTestUtils.java

```diff
@@ -0,0 +1,106 @@
+package io.flutter.util;
+
+import com.intellij.openapi.Disposable;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.application.PathManager;
+import com.intellij.openapi.module.Module;
+import com.intellij.openapi.roots.impl.libraries.ApplicationLibraryTable;
+import com.intellij.openapi.roots.libraries.Library;
+import com.intellij.openapi.util.Disposer;
+import com.intellij.openapi.util.io.FileUtil;
+import com.intellij.openapi.vfs.newvfs.impl.VfsRootAccess;
+import com.intellij.testFramework.ThreadTracker;
+import com.intellij.util.PathUtil;
+import com.jetbrains.lang.dart.sdk.DartSdk;
+import com.jetbrains.lang.dart.sdk.DartSdkGlobalLibUtil;
+import com.jetbrains.lang.dart.sdk.DartSdkUtil;
+import org.jetbrains.annotations.NotNull;
+import org.junit.Assert;
+
+import java.io.File;
+import java.util.Collections;
+
+// TODO(devoncarew): We should pare this class down, and move some of its functionality into FlutterTestUtils.
+// From //intellij-plugins/Dart/testSrc/com/jetbrains/lang/dart/util/DartTestUtils.java
+public class DartTestUtils {
```

Comment:
This should be in the com.jetbrains.lang.dart package with the other copied classes (which I'm about to commit).